src/main/java/mate/academy/spring/dao/RoleDao.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.spring.dao;
+
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.RoleName;
+
+public interface RoleDao {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```